.ci/magic-modules/generate-terraform.sh

```diff
@@ -29,6 +29,7 @@ bundle exec compiler -p products/redis -e terraform -o "${GOPATH}/src/github.com
 
 # Resources that were already using beta APIs before they started being autogenerated
 bundle exec compiler -v beta -p products/compute -t Address -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
+bundle exec compiler -v beta -p products/compute -t Subnetwork -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
```

Comment:
nit: -t takes a list, so you should be able to just do `-t Address,Subnetwork` (probably should verify this for sure though)